atacama.ipynb

```diff
@@ -0,0 +1,806 @@
+{
```

Comment:
I don't know if you decided to skip mentioning that this tutorial is using API instead of CLI, but it is worth mentioning. Depending on where the tutorials will be published in the end and in what "vicinity".